client/components/footer.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import { replaceComponent } from "/imports/plugins/core/components/lib";
+import { Reaction } from "/client/api";
+
+const FooterBeesKnees = () =>
```

Comment:
When doing stateless components with arrow functions always wrap in either a `()` or `{}` depending on your needs.

Reply:
done, like this:
```
const FooterBeesKnees = () => (
```
is this what you meant?